src/main/jhipster/src/main/webapp/app/entities/feedback/feedback-dialog.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Inject, Input, OnInit, Renderer2 } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { DOCUMENT } from '@angular/common';
+import { ParamContext } from '../../shared/service/param.context';
+import { FEEDBACK_SURVEY_ID } from '../../app.constants';
+import { FeedbackFeatureEnum } from './feedback-feature.enum';
+import { FeedbackService } from '../../shared/feedback/service/feedback.service';
+import { finalize } from 'rxjs/operators';
+import { AlertService } from '../../shared/alert/alert.service';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+    selector: 'jhi-feedback-dialog-component',
+    templateUrl: './feedback-dialog.component.html',
+    styleUrls: ['./feedback-dialog.component.scss']
+})
+export class FeedbackDialogComponent implements OnInit {
+
+    @Input() feature: FeedbackFeatureEnum;
+
+    ratingText: string;
+    dontShowAgain: boolean;
```

Comment:
alternatively it could go into `shared/feedback/components`

Reply:
Done